extensions/scarb-doc/src/main.rs

```diff
@@ -87,7 +88,7 @@ fn main_inner(args: Args, ui: Ui) -> Result<()> {
             }
         }
     }
-
+    ui.print(Status::new("Saved to directory:", output_dir.as_str()));
```

Comment:
imo, path `.../target/doc`  should be informative enough but let me know if you disagree (why) and think that the entire path `.../target/doc/my_project/output.json` file or `.../target/doc` respectively for user chosen output-format should be printed out.